### PR TITLE
Add nfts ui

### PR DESCRIPTION
**Pull request by AtlasSchmidt**
## create user interface ,
- add nfts blog component.
- add search bar

### CODE REVIEW

File: src/components/Search/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                value={str}
+                onChange={(e) => { setStr(e.target.value) }}
+            />
+            <IconButton type="button" sx={{ p: '10px' }} aria-label="search">
+                <SearchIcon />
+            </IconButton>
+            <Divider sx={{ height: 28, m: 0.5 }} orientation="vertical" />
+        </Paper>
+    );
+}
```
Comment (codeLover1126) [inline src/components/Search/index.js]: Make sure all files **must** have a blank line at end of the file.

File: src/pages/Nfts/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            <Box sx={{ display: 'flex', justifyContent: "center", marginTop: '90px' }}>
+                <Search />
+            </Box>
+            <Box sx={{ display: 'flex', justifyContent: "center", marginTop: '90px' }}>
+            </Box>
+        </Container>
+    )
+};
+
+export default Nfts;
```
Comment (codeLover1126) [inline src/pages/Nfts/index.js:15]: Missing the blank line here.

File: src/pages/Nfts/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React, { useEffect, useState } from "react";
+import Search from "components/Search";
+import { Container, Typography, Box } from "@mui/material";
+
+const Nfts = () => {
+
```
Comment (codeLover1126) [inline src/pages/Nfts/index.js]: This is **the** useless blank line.

File: src/pages/Nfts/index.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    return (
+        <Container maxWidth="xl">
```
Comment (codeLover1126) [inline src/pages/Nfts/index.js]: The indentation of space should be 2, not 4.

File: src/pages/Nfts/index.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import React, { useEffect, useState } from "react";
+import Search from "components/Search";
+import { Container, Typography, Box } from "@mui/material";
+
+const Nfts = () => {
+    return (
+        <Box sx={{ bgcolor: '#bcaaa4',}}>
+        <Box sx={{ display: "flex", justifyContent: "center", paddingTop: '60px' }}>
```
Comment (codeLover1126) [inline src/pages/Nfts/index.js:8]: Wrong indentation.

File: src/components/Blog/index.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+import { Container, Grid, Box } from '@mui/material';
+import Paper from '@mui/material/Paper';
+import InputBase from '@mui/material/InputBase';
+import Divider from '@mui/material/Divider';
+import IconButton from '@mui/material/IconButton';
+import SearchIcon from '@mui/icons-material/Search';
+
+const Blog = () => {
+    return (
+        <div>
```
Comment (codeLover1126) [inline src/components/Blog/index.js:11]: The space indentation number is 2, not 4.

File: src/components/Blog/index.js [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+                        <SearchIcon />
+                    </IconButton>
+                    <Divider sx={{ height: 28, m: 0.5 }} orientation="vertical" />
+                </Paper>
+            </Box>
+            <Container maxWidth="xl">
+                <div id='blogdata'>
+                    <Grid container spacing={10}>
+                        {filterBlog.map(item => (
+                            <Grid item lg={3} md={4} sm={6} xs={12} >
```
Comment (codeLover1126) [inline src/components/Blog/index.js:32]: `key` attr is missing here.

File: src/components/Blog/index.js [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+                                        component="img"
+                                        image={item.img}
+                                        alt={item.bear}
+                                    />
+                                    <CardContent sx={{ display: 'flex', justifyContent: "space-between", backgroundColor: "#80cbc4" }}>
+                                        <Typography gutterBottom variant="p">
+                                            {item.collectionName}
+                                        </Typography>
+                                        <Typography variant="p">
+                                            {item.price}$
```
Comment (codeLover1126) [inline src/components/Blog/index.js:44]: the USD currency ($) is prefix, not suffix.

**Review by codeLover1126 — CHANGES_REQUESTED** [review]

**Comment by codeLover1126** [comment]
All commit messages should be different. To do that, please specify more details in each one.

File: src/components/Search/index.js (hunk 1 above)
Comment (AtlasSchmidt) [reply src/components/Search/index.js]: this is resolved

File: src/pages/Nfts/index.js (hunk 3 above)
Comment (AtlasSchmidt) [reply src/pages/Nfts/index.js]: this is resolved

File: src/pages/Nfts/index.js (hunk 4 above)
Comment (AtlasSchmidt) [reply src/pages/Nfts/index.js]: this is resolved

File: src/pages/Nfts/index.js (hunk 5 above)
Comment (AtlasSchmidt) [reply src/pages/Nfts/index.js:8]: this is resolved

File: src/components/Blog/index.js (hunk 6 above)
Comment (AtlasSchmidt) [reply src/components/Blog/index.js:11]: this is resolved

File: src/components/Blog/index.js (hunk 7 above)
Comment (AtlasSchmidt) [reply src/components/Blog/index.js:32]: this is resolved

File: src/components/Blog/index.js (hunk 8 above)
Comment (AtlasSchmidt) [reply src/components/Blog/index.js:44]: this is resolved

File: src/pages/Nfts/index.js (hunk 2 above)
Comment (AtlasSchmidt) [reply src/pages/Nfts/index.js:15]: this is resolved

**Pull request merged**